descriptors/ModuleDescriptor-template.json

```diff
@@ -4,7 +4,7 @@
   "provides": [
     {
       "id": "users",
-      "version": "15.2",
+      "version": "15.3",
```

Comment:
Please also change the version in the RAML file for this interface

Reply:
You're right. I've done that!